Gems/ROS2/Code/CMakeLists.txt

```diff
@@ -44,7 +45,13 @@ ly_add_target(
             Gem::PhysX.Static
 )
 
-target_depends_on_ros2_packages(${gem_name}.Static rclcpp builtin_interfaces std_msgs sensor_msgs nav_msgs urdfdom tf2_ros ackermann_msgs gazebo_msgs control_toolbox)
+find_package(urdfdom)
+target_link_libraries(${gem_name}.Static PUBLIC urdfdom::urdfdom_model)
```

Comment:
Editor target

Reply:
Nice catch @adamdbrw ! Adjusted and tested.